crates/noirc_frontend/src/hir/resolution/resolver.rs

```diff
@@ -1266,6 +1279,48 @@ impl<'a> Resolver<'a> {
         let module_id = self.path_resolver.module_id();
         module_id.module(self.def_maps).is_contract
     }
+
+    // TODO: write resolver tests for this
+    fn resolve_call_arg(&mut self, argument: Expression, call_expr_span: Span) -> ExprId {
+        match argument.kind {
+            // Only function calls should convert the Str literal to a FmtStr literal
+            ExpressionKind::Literal(Literal::Str(string)) => {
+                let re = Regex::new(r"\{([\S]+)\}").unwrap();
```

Comment:
same re unwrap -> expect

Reply:
maybe it's better to use [lazy_regex](https://docs.rs/lazy-regex/latest/lazy_regex/)?